### DISCUSSION TITLE
Disable video autoplay

### DISCUSSION THREAD

**Discussion by SkypLabs** [ente-io/ente#2305] [Enhancements]
Hi,

I would like to be able to disable video autoplay.

I personally find it annoying, but more than anything, I would like to stress its impact on resource consumption, which is something we should all act on. As such, I would recommend disabling it by default while providing an option to enable it in the settings.

Thanks

**Comment by DxTa** [comment]
Maybe this is relating to mobile. Nevertheless, maybe it's good to have option to choose. Because I do want this to be supported in desktop

**Reply by SkypLabs** [reply]
I'd like it to be implemented in both mobile and web apps.

**Reply by taivlam** [reply]
The autoplaying of videos occurs in both the desktop and mobile versions of Ente.

I think autoplaying should be disabled by default for new users.

**Comment by neuged** [comment]
Would very much like an option to disable this. 

Something to consider for mobile: Note that on mobile autostarting a video seems to stop other audio from playing in the background. So I cannot listen to a podcast/voice message and at the same time swipe through fotos in ente, when there are videos among the fotos. The audio stops and does not start again when swiping away from the video (witnessed on ios). I personally do not want the video to play at all, but at a minimum it would be nice if ente could only start a video if nothing else is playing at the same time.
